app/src/test/java/org/simple/clinic/help/HelpRepositoryTest.kt

```diff
@@ -1,128 +0,0 @@
-package org.simple.clinic.help
```

Comment:
Do we no longer need `HelpRepositoryTest`?

Reply:
In the next PR, this will be replaced by a Room database backed implementation which is very simple and does not need to be tested.